build.gradle

```diff
@@ -294,8 +291,8 @@ configure(javaCodeCheckedProjects) {
     /* Code quality */
     tasks.withType(Checkstyle) {
         reports {
-            xml.enabled true
-            html.enabled true
+            xml.required = true
```

Comment:
Changed the variable name in checkstyle.

Reply:
Did we pick up a newer version of Checkstyle somewhere?  I didn't see it in the diffs.

Reply:
Yes, the checkstyle plugin is packaged with gradle - so it was implicitly updated.